app/Http/Controllers/SquawkController.php

```diff
@@ -56,19 +56,20 @@ public function __construct(SquawkService $squawkService)
      */
     public function getSquawkAssignment(string $callsign) : JsonResponse
     {
-        try {
+        $assignment = $this->squawkService->getAssignedSquawk($callsign);
+        if (!$assignment) {
             return response()->json(
                 [
-                    'squawk' => $this->squawkService->getAssignedSquawk($callsign)->squawk(),
-                ]
-            )->setStatusCode(200);
-        } catch (SquawkNotAssignedException $exception) {
-            return response()->json(
-                [
-                    'message' => $exception->getMessage(),
+                    'message' => 'Assignment not found for BAW12AZ',
```

Comment:
Was this meant to be hardcoded as BAW12AZ?

Reply:
Nope!